### DISCUSSION TITLE
Bluetooth connection only

### DISCUSSION THREAD

**Discussion by cynthih** [j-kaltes/Juggluco#116] [Q&A]
From what I've read about Juggluco, it can connect to a US Libre 2 sensor using only Bluetooth.  Is this correct?  I don't have a NFC-capable phone and don't want to use Libreview.  Thanks.

**Comment by j-kaltes** [comment]
No, you need an NFC-enabled phone to start the sensor and receive information about the sensor so that Juggluco can establish a Bluetooth connection with the sensor. Once the sensor is scanned by Juggluco on a phone, it can send connection information to other phones and watches, allowing them to also establish a Bluetooth connection to the sensor.
For almost everyone, it is not an option to start each sensor via their mother's phone and send the sensor information to their own phone via a mirror connection.  See: https://www.juggluco.nl/Juggluco/mirror/index.html
